app/controllers/fullcalendar_engine/events_controller.rb

```diff
@@ -121,5 +122,13 @@ def determine_event_type
     def make_time_from_minute_and_day_delta(event_time)
       params[:minute_delta].to_i.minutes.from_now((params[:day_delta].to_i).days.from_now(event_time))
     end
+
+    def set_event_color(event)
+      if event.color == nil
+        '#3B91AD' #default color
+      else
+        event.color 
+      end
```

Comment:
Where would I include the default value? I had a look in `config/initializers/configuration.rb `, but if I understand it correctly, those values are passed to FullCalendar. The `default_color` value should not be passed to FullCalendar, right?